Zend/zend_execute_API.c

```diff
@@ -627,6 +627,10 @@ int _call_user_function_ex(zval *object, zval *function_name, zval *retval_ptr,
 {
 	zend_fcall_info fci;
 
+	if (EG(timed_out)) {
+		zend_timeout();
+	}
```

Comment:
This should be in zend_call_function(), which is the low-level API.

Reply:
Done.